src/backend/routers/conversation.py

```diff
@@ -496,42 +504,6 @@ async def list_files(
     return files_with_conversation_id
 
 
-@router.put("/{conversation_id}/files/{file_id}", response_model=FilePublic)
-async def update_file(
```

Comment:
I don't think updating files is used at all. To reduce code and overhead of testing it I've just removed it.

@knajjars to confirm?